Tests/Core/PrettyTests.swift

```diff
@@ -132,6 +132,27 @@ class PrettyTests: XCTestCase {
             when(true,  then: #"["mike": Cat(id: "mike", name: Optional("ポチ")), "tama": Cat(id: "tama", name: Optional("タマ"))]"#),
         ])
     }
+    
+    ///
+    /// Tuple
+    ///
+    func testString_Tuple() {
+        let tuple = (1, ("one", URL(string: "https://www.example.com/")!))
+        assert(to:
+            curry(pretty.string)(tuple))
+            .expect([
+                when(false, then: #"(1, ("one", https://www.example.com/))"#),
+                when(true, then: #"(1, ("one", URL("https://www.example.com/")))"#),
+            ])
+        
+        let labeledTuple = (vintage: 2019, region: "Chili", variety: Optional("Chardonnay"), taste: ["round", "smooth", "young"])
```

Comment:
I think a better test would be include unlabeled data 🍷 

e.g.  `(2019, "Chili", variety: Optional("Chardonnay"), taste: ["round", "smooth", "young"])`

Reply:
I adopt it in daf0d9d3892eb690843267b1465e9c9927c92018.